demisto_sdk/commands/common/update_id_set.py

```diff
@@ -133,8 +133,41 @@ def does_dict_have_alternative_key(data: dict) -> bool:
     return False
 
 
+def get_item_marketplaces(item_path: str, item_data: Dict = None, packs: Dict[str, Dict] = None) -> List:
+    """
+    Return the supporting marketplaces of the item.
+
+    Args:
+        item_path: the item path.
+        item_data: the item data.
+        packs: the pack mapping from the ID set.
+
+    Returns: the list of supporting marketplaces.
+    """
+
+    if not item_data:
+        file_type = Path(item_path).suffix
+        item_data = get_file(item_path, file_type)
+
+    # first check, check field 'marketplaces' in the item's file
+    marketplaces = item_data.get('marketplaces', [])  # type: ignore
+
+    # second check, check the metadata of the pack
+    if not marketplaces and 'pack_metadata' not in item_path:
+        pack_name = get_pack_name(item_path)
+        if packs:
+            marketplaces = packs.get(pack_name, {}).get('marketplaces', [MarketplaceVersions.XSOAR.value])
+        else:
+            marketplaces = get_mp_types_from_metadata_by_item(item_path)
+    else:
+        # default supporting marketplace
+        marketplaces = [MarketplaceVersions.XSOAR.value]
+
```

Comment:
@yaakovi 
if there is marketplaces defined - the if will go to the else and will return the default one

Reply:
@ilappe  In what scenario do you see that happening?

Reply:
good catch! fixing now

Reply:
@reutshal 
it was for example in case the item has ['marketplacev2'] in the marketplaces key